db/schema.rb

```diff
@@ -228,8 +232,8 @@
   end
 
   create_table "log_events", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
-    t.integer  "loggable_id"
     t.string   "loggable_type"
+    t.integer  "loggable_id"
```

Comment:
Not a big deal, but can you try to exclude column order changes like this in schema? We've never really figured out a good way of dealing with this.